projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -160,6 +160,10 @@ public function create_item_permissions_check( $request ) { // phpcs:ignore Vari
 			return new WP_Error( 'rest_forbidden', __( 'Sorry, you are not allowed to access this endpoint.', 'jetpack-scheduled-updates' ), array( 'status' => 403 ) );
 		}
 
+		if ( get_option( 'wpcom_is_staging_site' ) ) {
```

Comment:
Should we move the check before checking on the current plan so there's no extra check to the feature?